Microsoft.DotNet.Interactive/Events/InputRequested.cs

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Threading;
+using Microsoft.DotNet.Interactive.Commands;
+using Newtonsoft.Json;
+
+namespace Microsoft.DotNet.Interactive.Events
+{
+    public class InputRequested : KernelEventBase
+    {
+        [JsonConstructor]
+        internal InputRequested(string prompt, bool password)
+        {
+            Prompt = prompt;
+            Password = password;
+        }
+
+        public InputRequested(
+            string prompt,
+            bool password,
```

Comment:
this could be easier to understand as isPassword with default as false.

Reply:
Agree. Change the parameter name to `isPassword`.
Do you mean to rename the property `InputRequested.Password` too? If that's the case, would you want me to add `[JsonProperty("Password")]` to that property? (I notice `KernelEventBase` sub-types all have `JsonConstructor`, but not sure if they really got serailized/deserialized anywhere).